qiskit/aqua/utils/circuit_cache.py

```diff
@@ -176,27 +177,26 @@ def load_qobj_from_cache(self, circuits, chunk, run_config=None):
                 # Need the 'getattr' wrapper because measure has no 'params' field and breaks this.
                 if not len(getattr(compiled_gate, 'params', [])) == len(getattr(uncompiled_gate, 'params', [])) or \
                     not compiled_gate.name == uncompiled_gate.name:
-                    raise AquaError('Gate mismatch at gate {0} ({1}, {2} params) of circuit against '
-                                         'gate {3} ({4}, {5} params) '
-                                         'of cached qobj'.format(cache_index, uncompiled_gate.name, len(uncompiled_gate.params),
-                                                                 gate_num, compiled_gate.name, len(compiled_gate.params)))
+                    raise AquaError('Gate mismatch at gate {0} ({1}, {2} params) of circuit against gate {3} ({4}, '
+                                    '{5} params) of cached qobj'.format(cache_index,
+                                                                 uncompiled_gate.name,
+                                                                 len(uncompiled_gate.params),
+                                                                 gate_num,
+                                                                 compiled_gate.name,
+                                                                 len(compiled_gate.params)))
                 compiled_gate.params = np.array(uncompiled_gate.params, dtype=float).tolist()
         exec_qobj = copy.copy(self.qobjs[chunk])
         if self.skip_qobj_deepcopy: exec_qobj.experiments = self.qobjs[chunk].experiments[0:len(circuits)]
         else: exec_qobj.experiments = copy.deepcopy(self.qobjs[chunk].experiments[0:len(circuits)])
 
-        if not run_config: run_config = {}
```

Comment:
there is another field, `seed`, I think we should store back to the qobj as well to assure the simulator use the same seed.